app/gen-server/migration/1713186031023-UserLastConnection.ts

```diff
@@ -0,0 +1,16 @@
+import {MigrationInterface, QueryRunner, TableColumn} from 'typeorm';
+
+export class UserLastConnection1713186031023 implements MigrationInterface {
+
+  public async up(queryRunner: QueryRunner): Promise<any> {
+    await queryRunner.addColumn('users', new TableColumn({
+      name: 'last_connection_at',
+      type: "date",
```

Comment:
In `FirstLogin1569593726320` we are adding `first_login_at` as `datetime` column using following snippet.

```
    const sqlite = queryRunner.connection.driver.options.type === 'sqlite';
    const datetime = sqlite ? "datetime" : "timestamp with time zone";
    await queryRunner.addColumn('users', new TableColumn({
      name: 'first_login_at',
      type: datetime,
      isNullable: true
    }));
    ```

Can we reuse this pattern here? Or is there any other reason to use just `date` column?

Reply:
Because we don't want to know the time of the last connection, only know last date the user connect it's fine, and we will updated it maximum once a day

Reply:
Since Sqlite doesn't have native `date` column (and all tests are using sqlite), can add some explanation (and comments) here how that works (what is actually created in each column, what is actually stored)? Or maybe update the `app/gen-server/lib/values.ts` with proper mapping between those two databases.